src/main/python/image_preprocessing/face_manager.py

```diff
@@ -25,6 +25,30 @@ def __init__(self, shape_predictor_path):
         """
         self.shape_predictor_path = shape_predictor_path
 
+    def face_likeness_extraction(self, image):
+        """
+        This function find a face in the image passed and is optimised
+        to align the face before being returned.
+        Author(s):
+            Stephan Nell
+        Args:
+            image (:obj:'OpenCV image'): Image containing the face we need to detect and extract.
+        Raises:
+            ValueError: If no face can be detected.
+        Returns:
+            obj:'OpenCV image': Any background and unnecessary components are removed and only
+            the aligned face is returned in gray scale for facial likeness.
+        Todo:
+            Return error if no face detected
+
+        """
+        rectangle = self.detect(image)
+        predictor = dlib.shape_predictor(self.shape_predictor_path)
+        face_aligner = FaceAligner(predictor, desiredFaceWidth=256)
+        face_aligned = face_aligner.align(image, image, rectangle)
+        gray_face = cv2.cvtColor(face_aligned, cv2.COLOR_BGR2GRAY)
```

Comment:
I don't think this line should be in this function, because it may be possible somewhere that the user that uses this function would like the face to still be in colour. It is also not prudent to assume that the image is in colour in this function. Rather move the line to another place (see comment that refers to this).